app/Services/Manager.php

```diff
@@ -189,19 +204,31 @@ public function collectContestInfo($data)
      *
      * @return object $signups
      */
-    public function getUserSignup($id, $campaign = null, $campaignRun = null)
+    public function getUserSignup($id, $campaign = null, $campaignRun = null, $getMultiple = false)
     {
         $signups = $this->northstar->getUserSignups($id, $campaign);
 
+        if ($getMultiple) {
+            $multipleSignups = [];
+        }
```

Comment:
@sbsmith86 yeah, good call. I think part of my hangup is in the naming and the numerous things it's trying to accomplish in one go.